pulsar-testclient/src/main/java/com/yahoo/pulsar/proxy/socket/client/PerformanceClient.java

```diff
@@ -62,17 +62,17 @@
         @Parameter(names = { "--conf-file" }, description = "Configuration file")
         public String confFile;
 
-        @Parameter(names = { "-u", "--proxy-url" }, description = "Pulsar Proxy URL", required = true)
+        @Parameter(names = { "-u", "--proxy-url" }, description = "Pulsar Proxy URL, e.g., \"ws://localhost:8080/\"", required = true)
         public String proxyURL;
 
         @Parameter(description = "/persistent/my-property/cluster1/my-ns/my-topic", required = true)
-        public String destination;
+        public List<String> destinations;
```

Comment:
JCommander did't support a single main parameter until it was update v1.72.
https://github.com/cbeust/jcommander/issues/90

Unfortunately, v1.72 is not in Maven Repository now.
https://mvnrepository.com/artifact/com.beust/jcommander

That's why, we have to define a main parameter as a `List<String>`.